command/pin.go

```diff
@@ -52,7 +52,9 @@ func (c *PinCommand) Flags() *flag.FlagSet {
 
 	f.Int64Var(&c.flagConcurrency, "concurrency", concurrency.DefaultConcurrency(1),
 		"maximum number of concurrent resolutions")
-	f.StringVar(&c.flagParser, "parser", "actions", "parser to use")
+	f.StringVar(&c.flagParser, "parser", "actions", 
+		fmt.Sprintf("parser to use [%s]", 
```

Comment:
I'm worried this could get lengthy over time as we add more parsers. Could we add it to the top-level help description as a list instead?

Reply:
Do you mean moving it into the `pinCommandHelp`?

Reply:
Could be something like this?
```
...
var pinCommandHelp = `
The "pin" command resolves and pins any unpinned versions to their absolute or
hashed version for the given input file:

    actions/checkout@v3 -> actions/checkout@2541b1294d2704b0964813337f...

The original unpinned version is preserved in a comment, next to the pinned
version. If a version is already pinned, it does nothing.

To update versions that are already pinned, use the "update" command instead.

Allowed parsers: %s

EXAMPLES

  ratchet pin ./path/to/file.yaml

FLAGS

`

...

func (c *PinCommand) Flags() *flag.FlagSet {
	f := flag.NewFlagSet("", flag.ExitOnError)
	f.Usage = func() {
		fmt.Fprintf(os.Stderr, "%s\n\n", strings.TrimSpace(
			fmt.Sprintf(pinCommandHelp, strings.Join(parser.List(), ", "))))
		f.PrintDefaults()
	}
...
```

Reply:
Sorry - it'll be easier if I open the PR to show what I mean.